conans/client/rest/conan_requester.py

```diff
@@ -17,6 +18,15 @@
 # TODO: Fix this security warning
 logging.captureWarnings(True)
 
+class SSLContextAdapter(HTTPAdapter):
+    def init_poolmanager(self, *args, **kwargs):
+        context = create_urllib3_context()
+        try:
+            context.load_default_certs()
+            kwargs['ssl_context'] = context
+        except AttributeError:
```

Comment:
The `try/except` block should catch both exceptions and raise a Conan one:

```python
try:
    from requests.packages.urllib3.util.ssl_ import create_urllib3_context
    context = create_urllib3_context()
    context.load_default_certs()
except (ImportError, AttributeError) as e:
    raise ConanException("....")
```

The exception message could be something like (just a suggestion): _"Conan cannot use SSL certificates from system stores, your `requests` and/or `urllib3` libraries doesn't support this functionality (you can deactivate it with 'conan config set general.use_system_certs=0')"_